src/cloudAdapter/genezio/genezioAdapter.ts

```diff
@@ -207,7 +208,11 @@ export class GenezioCloudAdapter implements CloudAdapter {
     ): Promise<string> {
         const finalStageName =
             stage != "" && stage != "prod" ? `-${stage.replaceAll(/[/_.]/gm, "-")}` : "";
-        const finalSubdomain = frontend.subdomain + finalStageName;
+        let finalSubdomain = frontend.subdomain + finalStageName;
+        if (finalSubdomain.length > 63) {
+            debugLogger.debug("Subdomain is too long. Generating random subdomain.");
+            finalSubdomain = frontend.subdomain?.substring(0, 56) + "-" + randomString(6);
```

Comment:
If I deploy the same frontend a second time, will I get a new URL?

Reply:
auch, you are right. that's an error on my side. I'll have to think of a way to ensure the same suffix is generated. maybe a hash of the stage name?

Reply:
Yea, a hash sounds like a good idea.